src/platform/apollolake/include/platform/memory.h

```diff
@@ -138,20 +138,24 @@
 
 /* Heap section sizes for system runtime heap for master core */
 #define HEAP_SYS_RT_0_COUNT64		64
-#define HEAP_SYS_RT_0_COUNT512		16
-#define HEAP_SYS_RT_0_COUNT1024		4
+#define HEAP_SYS_RT_0_COUNT128		16
+#define HEAP_SYS_RT_0_COUNT192		8
+#define HEAP_SYS_RT_0_COUNT256		4
+#define HEAP_SYS_RT_0_COUNT384		2
```

Comment:
Why are you adding 384 when it's the buffer block size ?

Reply:
Now the algorithm is:
````
   if (size >= 384)
        ptr =  rballoc
   else
          ptr = _malloc

    if ( !ptr)
           ptr = rballoc
````
Without 384 block, for the allocation 256 < size < 384 can't be supported by _malloc but wastes cpu time in _malloc.  So I add this to fix it

Reply:
There is opportunity now (since we have save memory by deleting 512 and 1024 blocks) to add more step sizes here. You could also add 320 block size. It does need a macro though for your largest block size before rballoc blocks for the logic statement above.